app/build.gradle

```diff
@@ -8,12 +8,13 @@ plugins {
 
 android {
     namespace 'com.nlinterface'
-    compileSdk 33
+    compileSdk 34
 
     defaultConfig {
         applicationId "com.nlinterface"
         minSdk 24
-        targetSdk 33
+        //noinspection EditedTargetSdkVersion
+        targetSdk 34
```

Comment:
Änderung der Compile SDK finde ich in Ordnung, aber die TargetSDK sollte erst nach längerer Review angepasst werden. Da gibts ja in Android Studio auch einen Guide und Checker dafür.